### DISCUSSION TITLE
figure.liquid sets nonstandard <img> attributes (eg max-width)

### DISCUSSION THREAD

**Discussion by renauddetry** [alshedivat/al-folio#2156] [Q&A]
`figure.liquid` generates an `<img>` tag that contains nonstandard attributes, like `max-width`.

When setting the value of these attributes (via liquid include args), I don't see the expected result. For instance, with `max-width` set to 100px, my figure still occupies the whole width. I don't understand what the role of these attributes is:
- I can't find them used anywhere in the project,
- browsers don't seem to pay attention to them.

To reproduce: In `2_project.md`, paste this line anywhere:

`{% include figure.liquid path="assets/img/1.jpg" title="example image" class="img-fluid rounded z-depth-1" max-width="100px" %}`

I would expect the image to be 100px wide. It's not.

**Comment by george-gca** [comment]
What do you mean by nonstandard? At least it is documented in the [W3 Schools](https://www.w3schools.com/cssref/pr_dim_max-width.php) as such.

I thought that the problem was that you are also using the `img-fluid` class, but removing it doesn't solve it. Idk why it is not behaving as expected.

**Reply by renauddetry** [reply]
Yes, `max-width` is a CSS property, but it doesn't seem to be a [valid attribute of the img tag](https://www.w3schools.com/tags/tag_img.asp).

Apologies if I'm missing something.

PS: thank you for al-folio, it's fantastic!!
